utils/docker/Dockerfile.centos.7

```diff
@@ -66,7 +66,7 @@ RUN yum -y install epel-release; \
 RUN yum -y install \
            python2-avocado python2-avocado-plugins-output-html \
            python2-avocado-plugins-varianter-yaml-to-mux       \
-           yum-plugin-copr;                                    \
+           yum-plugin-copr python2-clustershell;               \
```

Comment:
Shall we fix the other Dockerfile too?

Reply:
I am not sure may be @brianjmurrell can comment. Avocado related package only in centos7 so may be because we have Pipeline running only on system which has CentOS.

Reply:
This change was made for linting, which we only do on EL7.

Adding clustershell to the test nodes was done in daos-stack/pipeline-lib#55 but again only for EL7 since that is the only platform we are running Functional testing on.